docs/QUICK_START.md

```
@@ -99,49 +99,205 @@ sudo systemctl enable postgresql
+
+# Run migrations with uv run
+uv run alembic upgrade head
+
+# Navigate back to backend directory
+cd ..
+
+# Start the backend server
+uv run uvicorn main:app --reload --host 0.0.0.0 --port 8000
+```
+
+**Option B: Activate Virtual Environment First**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)